Git Formats/Git Config.sublime-syntax

```diff
@@ -211,7 +211,7 @@ contexts:
           pop: true
 
   line-end:
-    - match: $\n?
+    - match: (?=$)
```

Comment:
For the record, this is equivalent to `$`.

Reply:
A good point, thanks.